bftengine/src/bcstatetransfer/DBDataStore.cpp

```diff
@@ -28,7 +28,7 @@ std::string toString(const DataStore::CheckpointDesc& desc) {
 void DBDataStore::load(bool loadResPages_) {
   LOG_DEBUG(logger(), "");
   if (!get<bool>(Initialized)) {
-    LOG_INFO(logger(), "Not initialized");
+    LOG_INFO(logger(), " DBDataStore Not initialized");
```

Comment:
" DBDataStore Not initialized" -> "DBDataStore is not initialized"

Reply:
updated changes